.ci/scripts/docker-test.sh

```diff
@@ -15,7 +15,7 @@ fi
 # Skip for node v8 because it results in this warning:
 #   openssl config failed: error:25066067:DSO support routines:DLFCN_LOAD:could not load the shared library
 if [[ $major_node_version -gt 8 ]]; then
-  export NODE_OPTIONS="${NODE_OPTIONS:+${NODE_OPTIONS}} --openssl-config=./test/openssl-config-for-testing.cnf"
+  export NODE_OPTIONS="${NODE_OPTIONS:+${NODE_OPTIONS}} --openssl-config=$(pwd)/test/openssl-config-for-testing.cnf"
```

Comment:
REVIEW NOTE:  This was necessary to avoid this error message:
```
openssl config failed: error:02001002:system library:fopen:No such file or directory
```

when executing a subprocess to run `node fixtures/use-s3.js` as part of `s3.test.js` below... because that subprocess is run in a separate directory (in "test/instrumentation/modules/aws-sdk") rather than at the top-level. Using an absolute path fixes this -- which is the same thing that is done for this in test/test.js:

```
args.unshift('--require', path.join(__dirname, '_promise_rejection.js'))
```